GitUI/CommandsDialogs/FormCommandlineHelp.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="_NO_TRANSLATE_commands.Text" xml:space="preserve">
-    <value>browse [path] [-filter=] [-commit=]
+    <value>browse [path] [-filter=] [-commit=&lt;sha1&gt;[,&lt;sha2&gt;]]
```

Comment:
Unfortunately, the argument option `-filter=` from <= 2012 is inconsistent with most of the others.
So rather?
```suggestion
    <value>browse [path] [-filter=] [--commit &lt;sha1&gt;[,&lt;sha2&gt;]]
```

Reply:
-commit is also an old option, cannot be changed (probably used from shell extension too)

Reply:
I think `--commit=abc123,09982039cbs00808dsfsd08978s` is totally fine

Reply:
If --commit (or --selected-commit and --first-commit) is added then -commit should be handled still. Maybe shell extension could be synchronized (not for me that only run private builds...) but vsix and other users could be affected, I do not see all risks. It is hard to change published interfaces.

Reply:
This `[-commit=&lt;sha1&gt;[,&lt;sha2&gt;]]` looks like an additive change - i.e. the original functionality should be unaffected. We just allow to append another hash to it.
Am I overlooking something else?

Reply:
Modifying arguments for -commit is compatible.
@mstv remarked on the other arguments using GNU/Posix standard that single char options have one dash and can often be combined (ls -ar is the same as ls -a -r) and multicharacter options has two dashes (that cannot be combined) (ls --all --reverse).

Reply:
> @mstv remarked on the other arguments using GNU/Posix standard that ... and multicharacter options has two dashes.

But I missed that `-commit=...` has already existed.
Since GE is mainly a GUI tool, I think a cleanup of the sligthly inconsistent format of the CLI arguments is not worth to be cleaned up.